packages/next/src/server-build.ts

```diff
@@ -291,6 +292,35 @@ export async function serverBuild({
       // if the trace is unavailable we trace inside the runtime
     }
 
+    try {
+      const possiblePaths = [
+        path.join(
+          entryPath,
+          outputDirectory,
+          'server',
+          'instrumentation.js.nft.json'
+        ),
+        path.join(
+          entryPath,
+          outputDirectory,
+          'server',
+          'src',
+          'instrumentation.js.nft.json'
```

Comment:
Why is this nested under `server/src` in some cases? We should fix this on the Next.js side to be consistent.